deployments/Dockerfiles/autonomy-user/requirements.txt

```diff
@@ -1,4 +1,4 @@
-open-autonomy[all]==0.3.2
+open-autonomy[all]==0.3.4
```

Comment:
Need to update the release process, issue for the same https://github.com/valory-xyz/open-autonomy/issues/1512